### PR TITLE
epxanding / collapsing package search toolbar

### PR DESCRIPTION
**Pull request by derickbailey**
this pull request adds a Packages Search box to the main site layout and adjusts the old site layout's search box at the same time.  the search boxes now use css to expand and collapse

### CODE REVIEW

File: chocolatey/Website/Views/Shared/OldLayout.cshtml [hunk 1]
```diff
@@ -96,12 +96,10 @@
                 <li><a href="https://groups.google.com/group/chocolatey">Forum*</a></li>
                 <li><a href="http://cafepress.com/chocolatey">Shop*</a></li>
             </ul>
-            <div id="searchBox" role="search">
-                <form action="@Url.PackageList()" method="get">
-                <input name="q" id="searchBoxInput" title="Search by id only with 'id:searchValue'. Search by tags with 'tag:searchValue'." placeholder="Search Packages" value="@(String.IsNullOrEmpty(ViewBag.SearchTerm) ? "" : ViewBag.SearchTerm)" autofocus />
```
Comment (ferventcoder) [inline chocolatey/Website/Views/Shared/OldLayout.cshtml]: This means no more auto-focus. I'm okay with that.

Comment (derickbailey) [reply chocolatey/Website/Views/Shared/OldLayout.cshtml]: right - i meant to ask you about that. the auto-focus causes the full search box to be expanded as soon as the page loads, and covers the entire menu. not a good ux. it's much nicer with auto focus gone

Comment (ferventcoder) [reply chocolatey/Website/Views/Shared/OldLayout.cshtml]: No worries.

**Comment by ferventcoder** [comment]
Sometimes the best way to get feedback is to move fast - #543 #544 

We can easily rollback most of the time (which I've already done).

**Pull request merged**